db/models.py

```diff
@@ -43,7 +43,7 @@ class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(CinemaHall,
                                     on_delete=models.CASCADE,
-                                    related_name="movies")
+                                    related_name="sessions")
     movie = models.ForeignKey(Movie,
                               on_delete=models.CASCADE,
                               related_name="sessions")
```

Comment:
We don't write code in this way. Each parameter should be added on the next line

Reply:
Fix everywhere